cms/djangoapps/contentstore/views/course.py

```diff
@@ -810,6 +810,15 @@ def _create_or_rerun_course(request):
                     status=400
                 )
 
+        # Allow user to create the course only if they belong to the organisation
+        if settings.FEATURES.get('RESTRICT_NON_ORG_COURSE_CREATION'):
+            has_org_permission = bool(auth.get_user_permissions(request.user, None, org))
+            if not has_org_permission:
+                return JsonResponse(
+                        {'error': _('User is not present in the organisation')},
```

Comment:
@farhaanbukhsh, we may have to update this message after edX's review. At this point, I'd suggest something like

```
User does not have the permission to create courses in this organization
```

Reply:
that does sound reasonable. Thanks :smile: